docs/src/pages/guides/status-notification.md

```diff
@@ -0,0 +1,32 @@
+---
+id: status-notification
+title: Status Notification
+---
+
+Monika sends status notification based on a schedule with following information:
```

Comment:
Monika sends status notification periodically

Reply:
updated in https://github.com/hyperjumptech/monika/pull/436/commits/46c8e3f73eb09beceea103feb1c941afc4be6a3f